packages/components/src/components/page-error/page-error.jsx

```diff
@@ -63,6 +87,8 @@ PageError.propTypes = {
     messages: PropTypes.array,
     redirect_label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
     redirect_url: PropTypes.string,
+    setError: PropTypes.func,
+    clear_error: PropTypes.bool,
```

Comment:
can we rename this to `should_clear_error` to follow the guideline?
Boolean variables should start with `is`, `has`, `should`, ...

Reply:
since this happens on click can we name it to `should_clear_error_on_click`?